### PR TITLE
Change Server/cmdline Tutorial to use new `dart` tool

### PR DESCRIPTION
**Pull request by shubhanshu02**
Fixes part of #2617.

#### Changes Proposed
Changed `src/_tutorials/server/cmdline.md`(https://dart.dev/tutorials/server/cmdline) to use the new `dart` tools.

@kwalrath please review these changes.

### CODE REVIEW

File: src/_tutorials/server/cmdline.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ command-line arguments, files and directories, and more.
 
 ## Running an app with the standalone Dart VM
 
-To run a command-line app, you need the Dart VM (`dart`),
+To run a command-line app, you need the Dart command (`dart run`),
 which comes when you [install the Dart SDK](/get-dart).
```
Comment (kwalrath) [inline src/_tutorials/server/cmdline.md:51]: I'd like to tweak this a bit and change the SDK link:

```suggestion
To run a command-line app in the Dart VM, use `dart run`.
The `dart` command is part of the [Dart SDK](/tools/sdk).
```

File: src/_tutorials/server/cmdline.md [hunk 2]
```diff
@@ -96,7 +95,7 @@ brief description of key app features, click the highlighted code below.
 
 You might notice that dcat depends on a package named **args**.
 To get the args package, use the
-[pub package manager](/guides/packages).
+[dart pub package manager](/guides/packages).
```
Comment (kwalrath) [inline src/_tutorials/server/cmdline.md:98]: It's still the pub package manager; only the command needs the `dart` preface.

```suggestion
[pub package manager](/guides/packages).
```

**Review by kwalrath — COMMENTED** [review]
Looks great! Just a couple of tweaks.

File: src/_tutorials/server/cmdline.md (hunk 1 above)
Comment (shubhanshu02) [reply src/_tutorials/server/cmdline.md]: Done

File: src/_tutorials/server/cmdline.md (hunk 2 above)
Comment (shubhanshu02) [reply src/_tutorials/server/cmdline.md]: Okay, thanks for the clarification.

**Review by kwalrath — APPROVED** [review]
Thank you, @shubhanshu02!

**Pull request merged**